Localization/StringsConvertor/input/Base.lproj/app.json

```diff
@@ -620,7 +620,7 @@
             "intro": "These are the posts gaining traction in your corner of Mastodon."
         },
         "favorite": {
-            "title": "Your Favorites"
+            "title": "Favorites"
```

Comment:
Did you change that in Crowdin, too? Otherwise all of those changes to translated `app.json` will be overwritten.

Reply:
Will do!

Reply:
They don’t seem to have a translation for English (regular or US) in Crowdin, but I can monitor after merge to see if the changes get reverted.

Reply:
Okay, this seems to be a misunderstanding, sorry for that :) I meant basically every other `app.json` in this PR except the `Base.lproj/app.json`:

Your changes in `da.lproj/app.json` will be overwritten. The change in `he.lproj/app.json`, too, and so on and so forth. Does it make more sense now?

Reply:
Those translations also appear to be blank in the Crowdin UI, e.g. for Hebrew:
<img width="730" alt="Screenshot_2023-02-14 16 14 21" src="https://user-images.githubusercontent.com/25517624/218864547-9cbac376-f578-4762-92d3-275ed0a009f2.png">

I imagine they’ll update to match the base string (and I don’t want to contribute an English “translation” as that would be wrong)

Reply:
Ah, okay, I think, now I see your point. Took me a second, sorry 🙈 

If there are no translations, the base-strings will be used for those untranslated texts. As you changed the base-string, this will be used and your changes for Hebrew, for example, will be changed from "Favorites" to "Favorites". So in the end you did something, which Crowdin does automatically, but as you changed it to the same outcome, nothing changes :D